source/jormungandr/jormungandr/scenarios/ridesharing/blablalines.py

```diff
@@ -136,9 +138,13 @@ def _make_response(self, raw_json):
                     type_pb2.GeographicalCoord(lon=res.dropoff_place.lon, lat=res.dropoff_place.lat)
                 )
 
-            res.price = offer.get('price', {}).get('amount')
             currency = offer.get('price', {}).get('currency')
-            res.currency = "centime" if currency == "EUR" else currency
+            if currency == "EUR":
+                res.currency = "centime"
+                res.price = offer.get('price', {}).get('amount') * 100.0
```

Comment:
Is `res.price` a float or an int ? 
In other words, is a cast to int needed ?

Reply:
Int or float, once multiplied with a float will change it to float. The price is in float in navitia.

Reply:
Allright, everything's good then :+1: